torch/csrc/jit/passes/python_print.cpp

```diff
@@ -590,8 +644,12 @@ struct PythonPrintPass {
     }
   }
 
-  bool isLongLine(const std::string& str) {
-    return str.size() + level * 2 >= 40;
+  bool isLongLine(const std::vector<TaggedToken>& expr) {
+    size_t s = 0;
```

Comment:
This would be cleaner if we just stored the tagged strings as a TaggedString class rather than a vector. It would be better to encapsulate it here rather than letting its internal implementation leak.